src/ApiCompat/Microsoft.DotNet.ApiCompatibility/Abstractions/ElementContainer.cs

```diff
@@ -1,14 +1,26 @@
 ﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+using Microsoft.CodeAnalysis;
+
 namespace Microsoft.DotNet.ApiCompatibility.Abstractions
 {
     /// <summary>
     /// Class to wrap an Element of T with it's <see cref="MetadataInformation"/>.
     /// </summary>
     /// <typeparam name="T">The type of the Element that is holded</typeparam>
-    public class ElementContainer<T>
+    public class ElementContainer<T> where T : ISymbol
```

Comment:
This type was missing a generic constraint. Element is intended to be of type ISymbol.